BeeSafe/beesafe-services
# Bs-Create Kafka

What I did:
- Split up api's into controllers that correlate to their business logic using the blueprint class to register these api's into the flask server
- Created a temporary producer file, that when gets executed get's consumed by Kafka inside of our flask server

How To Test:
- Get the flask server running
   - requires you to have Kafka working
   - Run the commands below in one terminal to run zookeeper:
```
cd <location_of_kafka>/
bin/zookeeper-server-start.sh config/server.properties
```
   - Run these commands below in another terminal to get the Kafka server running
```
cd <location_of_kafka>/
bin/kafka-server-start.sh config/server.properties
```
   - then finally run `python server.py` after Kafka and zookeeper are running
   - run python Consumer.py located under `KafkaServices` folder and then go to the flask terminal to see the messages being consumed
- Test at least one API from each new controller file to make sure the api's are working
